frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -73,6 +73,27 @@ export class TemplateValidations {
     });
   };
 
+  // Return a valid bus, return the defaultBus if it's valid.
+  getAValidBus = (defaultBus?: DiskBus): DiskBus => {
```

Comment:
```suggestion
  getAValidBus = (defaultBus = DiskBus.VIRTIO): DiskBus => {
```

could we use this approach?